qemu/tests/cfg/seabios.cfg

```diff
@@ -8,19 +8,50 @@
     boot_menu = on
     enable_sga = yes
     image_verify_bootable = no
-    boot_menu_key = "f12"
-    Host_RHEL.m7:
-        boot_menu_key = "esc"
-    boot_menu_hint = "Press .*(F12|ESC) for boot menu"
-    # Specify the boot device name which you want to test here.
-    boot_device = "iPXE"
-    Host_RHEL.m6:
-        boot_device = "gPXE"
-    # SGA Bios info message, using sep as ";"
-    # Please update this message to suit for your own system.
-    restart_key = "ctrl-alt-delete"
-    Host_RHEL:
-        sgabios_info = "Google, Inc.\s"
-        sgabios_info += "Serial Graphics Adapter .*\s"
-        sgabios_info += "SGABIOS \$Id: sgabios.S .*\s"
-        sgabios_info += "\d \d\s"
+    variants:
+      - reboot_time_test:
+        images += " stg"
+        image_name_stg = "images/storage"
+        image_format_stg = "qcow2"
+        drive_format_stg = virtio
+        image_size_stg = 4G
+        create_image_stg = yes
+        remove_image_stg = yes
+        bootindex_stg = 0
+        bootindex_image1 = 1
+        variants:
+          - time0:
+              reboottimeout = 0
+          - time1:
+              reboottimeout = 1000
+          - time2:
+              reboottimeout = 10000
+          - time3:
+              reboottimeout = 30000
+          - time4:
+              reboottimeout = 60000
+          - time5:
+              reboottimeout = 65535
+          - time6:
+              reboottimeout = 65536
+          - time7:
+              reboottimeout = -1
+      - sga_restart:
+        boot_menu_key = "f12"
+        Host_RHEL.m7:
+           boot_menu_key = "esc"
+        boot_menu_hint = "Press .*(F12|ESC) for boot menu"
```

Comment:
the common params is separated into several parts, it's better to put them together

Reply:
@suqinhuang Thanks, I putted some lines together.  please check it again.